### ISSUE TITLE
Implement Quality Score Revisions

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#298]
As per our forum post: https://forum.giveth.io/t/revise-quality-score-for-projects-on-giveth-io/104/20

It's time to revise the quality score! This is what we decided:

- A project gets points for a project description over 100 characters and a photo (max 80 points)
- +100 points for being verified
- +for donations… Score = sqrt ( amonut raised)
- +for likes… Score = sqrt (10 * number of likes)
- There’s no limit to the score, be we do a weighted shuffle or a threshold calculation to choose the top 3 featured on the homepage (devs can decide what makes the most sense)
- Add a decay function so every score loses 1% per day

We need to also implement this retroactively to projects that are already on the DApp... clear their current score and calculate a new one w the new structure.

@mateodaza

**Comment by laurenluz** [comment]
@MoeNick @mateodaza can we get this on the priority list... I really wish the project sorting was improved... the default right now are projects that were added a long time ago... and then new projects that gain popularity have no chance to catch up.

**Comment by MoeNick** [comment]
I understand the importance, but I think we have to rush the TP version with a new design to the end and after that, we start adding new functionalities. You know, it's too hard keeping the 2 versions running, so the sooner we finish the TP, the sooner to be able to add features and functionalities.

**Comment by laurenluz** [comment]
@MoeNick does the new design improve the sort & filter function? A user in the community call mentioned that it was quite a challenge to find good project... I think both the quality score and imrprovements to sort/filter are necessary for this but I made an issue before and it was mentioned that that was being addressed already

**Comment by MoeNick** [comment]
Yeah we have to have it in our roadmap, not planned for 1st new design version.
